crates/python/src/qpu/api.rs

```diff
@@ -421,7 +422,7 @@ impl PyExecutionOptions {
     fn api_options(&self) -> Option<PyApiExecutionOptions> {
         self.as_inner()
             .api_options()
-            .map(|x| PyApiExecutionOptions(x.clone().into()))
+            .map(|x| PyApiExecutionOptions((*x).into()))
```

Comment:
Was this suggested by `clippy`? It feels less explicit to me.

Reply:
Yes, but `x` implements Copy now, so this is different.